c_stubs/xenctrlext_stubs.c

```diff
@@ -210,6 +209,135 @@ CAMLprim value stub_xenctrlext_assign_device(value xch, value domid,
     CAMLreturn(Val_unit);
 }
 
+/* based on xenctrl_stubs.c */
+static int get_cpumap_len(value xch, value cpumap)
+{
+	int ml_len = Wosize_val(cpumap);
+	int xc_len = xc_get_max_cpus(_H(xch));
+
+	if (ml_len < xc_len)
```

Comment:
Could be also written as:
```
return (ml_len < xc_len ? ml_len : xc_len);
```

Reply:
this is `min(ml_len, xc_len)`

Reply:
Yes, but I couldn't find a MIN macro in any of the headers that we currently import.